devito/passes/iet/languages/CXX.py

```diff
@@ -0,0 +1,63 @@
+from devito.ir import Call, UsingNamespace
+from devito.passes.iet.langbase import LangBB
+
+__all__ = ['CXXBB']
+
+
+std_arith = """
```

Comment:
So, all of these will be printed even if no needed?
I mean it is clear that all of these are custom operations (), so probably needed.

I am not sure I totally understand, but just naive question.
Is division by zero handled?